.azure-pipelines-templates/common_test.yml

```diff
@@ -1,21 +1,18 @@
 jobs:
```

Comment:
General comment: when this PR is about to get merged, we should ensure that the release pipelines still works!

Reply:
One way to do that is to cut a 0.8.1 where the changelog is "Single build for both consensus implementations, which can now be selected with a runtime flag".